envctl/env_scripts/java/appengine_flex.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e # exit on any failure
+set -o pipefail # any step in pipe caused failure
+set -u # undefined variables cause exit
+
+
+SERVICE_NAME="log-java-flex-con-$(echo $ENVCTL_ID | head -c 10)"
+
+destroy() {
```

Comment:
please document this and other methods - mostly in terms of what and why is happening here and how it related to the lifecycle of env tests framework (so others who might start off with this code, can quickly adapt it to other use-cases

Reply:
good point, I opened https://github.com/googleapis/env-tests-logging/issues/64 to address this for all languages in a future PR